repositories/src/main/scala/cromwell/pipeline/datastorage/dto/Project.scala

```diff
@@ -45,3 +55,18 @@ object ProjectUpdateRequest {
 final case class Version(value: String) extends AnyVal
 
 final case class ProjectFile(path: Path, content: String)
+
+sealed abstract class Visibility(val value: String)
+case object Private extends Visibility("private")
+case object Internal extends Visibility("internal")
+case object Public extends Visibility("public")
+object Visibility {
+  implicit lazy val visibilityFormat: Format[Visibility] =
+    implicitly[Format[String]].inmap(Visibility.fromString, _.value)
+  def fromString(s: String): Visibility = s match {
+    case "private"  => Private
+    case "internal" => Internal
+    case "public"   => Public
+  }
+  def values = Seq(Private, Internal, Public)
+}
```

Comment:
It's better to implement `Visibility` as a sealed trait. Remove value. And create `toString` method. (as we did with `fromString`)

Reply:
I disagree with that, but fixed :)